tests/acceptance/test_plugins.py

```diff
@@ -55,6 +54,21 @@ async def test_multi_get_hit_miss_ratio(self, memory_cache, data, ratio):
             == len(hits) / memory_cache.hit_miss_ratio["total"]
         )
 
+    @pytest.mark.asyncio
+    @pytest.mark.parametrize(
+        "key,value,namespace", [("a", 1, None), ("b", 2, "test")],
+    )
+    async def test_set_and_get(self, memory_cache, key, value, namespace):
```

Comment:
the case of not having a namespace is already tested above so we are good for the first case. Let's leave only one test where we pass the namespace explicitly and then we are ready to merge :)